controllers/ptpinstance_controller.go

```diff
@@ -582,6 +582,22 @@ func (r *PtpInstanceReconciler) Reconcile(ctx context.Context, request ctrl.Requ
 		return reconcile.Result{}, err
 	}
 
+	// Restore the Ptp instance status
+	if r.checkRestoreInProgress(instance) {
+		r.ReconcilerEventLogger.NormalEvent(instance, common.ResourceUpdated, "Restoring '%s' Ptp Instance resource status without doing actual reconciliation", instance.Name)
+		if err := r.RestorePtpInstanceStatus(instance); err != nil {
+			return reconcile.Result{}, err
+		}
+		if err := r.ClearAnnotation(instance); err != nil {
+			return reconcile.Result{}, err
+		}
+		return ctrl.Result{}, nil
+	}
+
+	if instance.Status.ObservedGeneration == instance.ObjectMeta.Generation {
```

Comment:
Need to add `instance.Status.Reconciled ` just as in platformnetwork_controller.go.

```
	if instance.Status.ObservedGeneration == instance.ObjectMeta.Generation &&
		instance.Status.Reconciled {

```

Reply:
Changes are done